elements/lisk-validator/src/validation.ts

```diff
@@ -74,7 +74,7 @@ export const isHexString = (data: unknown): boolean => {
 		return false;
 	}
 
-	return data === '' || /^[a-f0-9]+$/i.test(data);
+	return data === '' || /^([0-9a-fA-F]{2})+$/i.test(data);
```

Comment:
There is no need to expand the regular expression with `A-F` pattern. It already includes `i` flag after the closing slash, which means that the pattern is case insensitive 🤓 

```
> /^([a-f0-9]{2})+$/i.test('c0FfEe')
true
```